### ISSUE TITLE
[dev.icinga.com #7977] Provide expected data format and example

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#1247]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/7977

**Created by mhein on 2014-12-03 12:03:12 +00:00**

Assignee: _jmeyer_
Status: _Resolved_  (closed on _2015-03-06 09:07:25 +00:00_)
Target Version: _2.0.0-beta3_
Last Update: _2015-03-06 09:07:25 +00:00 (in Redmine)_

---

http://www.w3.org/TR/WCAG20-TECHS/G89.html wrote:
> The objective of this technique is to help the user avoid input errors by informing them about restrictions on the format of data that they must enter. This can be done by describing characteristics of the format or providing a sample of the format the data should have. \[...\]

**Acceptance Criteria:**

-   Identify form controls that will only accept user input data in a given format.
-   Determine if each of the form controls identified in 1 provides information about the expected format.
